akka-actor-tests/src/test/scala/akka/io/dns/DnsSettingsSpec.scala

```diff
@@ -120,11 +134,41 @@ class DnsSettingsSpec extends AkkaSpec {
           resolve-timeout = 1s
           search-domains = "example.com"
           ndots = 5
+          positive-ttl = forever
+          negative-ttl = never
         """))
 
       dnsSettings.NDots shouldEqual 5
     }
 
+    "parse ttl" in {
+      val dnsSettingsNeverForever = new DnsSettings(
+        eas,
+        ConfigFactory.parseString("""
+          nameservers = "default"
+          resolve-timeout = 1s
+          search-domains = []
+          ndots = 1
+          positive-ttl = forever
+          negative-ttl = never
+        """))
+
+      dnsSettingsNeverForever.positiveCachePolicy shouldEqual CachePolicy.Forever
+      dnsSettingsNeverForever.negativeCachePolicy shouldEqual CachePolicy.Never
+
+      val dnsSettingsDuration = new DnsSettings(
+        eas,
+        ConfigFactory.parseString("""
```

Comment:
I think the good solution is adding a default inline `Config` in this test. And override specific settings in the each test case. What do you think about this?

Reply:
You can use `.withFallback(otherConfig)` for that.